src/server_soc_requirements.adoc

```diff
@@ -260,7 +260,17 @@ deliver external interrupts to the RISC-V application processor harts.
      RISC-V hart. The host bridge (also known as IO bridge) invokes the IOMMU
      for address translations. To perform the operations requested by the host
      bridge the IOMMU may need to access in-memory data structures such as the
-     device directory table and page tables._
+     device directory table and page tables.                                   +
+                                                                               +
+     The physical memory protection limit access from IOMMUs to phusical
```

Comment:
I suggest "physical memory protection limit access" => "PMPs limit access"

"phusical "is misspelled.

Reply:
Was trying to avoid using PMP term to avoid confusion with hart PMP. 
I have updated as "physical memory protection checks limit access" (added "checks").
Fixed the misspelt word.